src/Doc/Lexer.php

```diff
@@ -178,12 +178,14 @@ private function getAnyType($string)
             $string = substr($string, 1);
 
             $typeInfo = $this->scanNextType($string);
-
+            if ($typeInfo === null) {
+                break;
```

Comment:
This will not work because the loop should continue while $string still has content. Please check my latest commit on the master branch which should fix it. If this is fine for you, I will create a new version tag

Reply:
ah damn, true :D Yes, the code in current master looks fine :+1:

Reply:
ah damn, true :D Yes, the code in current master looks fine +1